src/configuration.ts

```diff
@@ -10,7 +10,7 @@ const config: GlobalConfig = {
 	TOKEN_NAME: 'DRGIV',
 	WEB3_POLLING_INTERVAL: 15000,
 	SUBGRAPH_POLLING_INTERVAL: 5000,
-	TOKEN_PRECISION: 4,
+	TOKEN_PRECISION: 2,
```

Comment:
@aminlatifi 
Lauren asked to change TOKEN_PRECISION to 4 and if it's lower than 0.0001 show <0.0001.

Reply:
@aminlatifi I fixed it